src/bridges/aave/lending/AaveLendingBridge.sol

```diff
@@ -0,0 +1,342 @@
+
+        IERC20Metadata aToken = IERC20Metadata(aTokenAddress);
+
+        string memory name = string(abi.encodePacked('ZK-', aToken.name()));
+        string memory symbol = string(abi.encodePacked('ZK-', aToken.symbol()));
+
+        address zkAToken = address(new AccountingToken(name, symbol, aToken.decimals()));
+
+        underlyingToZkAToken[underlyingAsset] = zkAToken;
+
+        emit UnderlyingAssetListed(underlyingAsset, zkAToken);
+    }
+
+    /**
+     * @notice Convert function called by rollup processor, will enter or exit Aave lending position
+     * @dev Only callable by the rollup processor when also satisfying asset sanity checks
+     * @param inputAssetA The input asset, accepts Eth or ERC20 if supported underlying or zkAToken
+     * @dev if inputAssetA is ETH, will wrap to WETH and supply to Aave
+     * @param inputAssetB Unused input asset, reverts if different from NOT_USED
+     * @param outputAssetA The output asset, accepts Eth or ERC20 if supported underlying or zkAToken
+     * @dev if outputAssetA is ETH, will unwrap WETH and transfer ETH
+     * @param outputAssetB Unused output asset, reverts if different from NOT_USED
+     * @param totalInputValue The input amount of inputAssetA
+     * @param interactionNonce The interaction nonce of the call
+     * @param auxData Unused auxiliary information
+     * @return outputValueA The output amount of outputAssetA
+     * @return outputValueB The ouput amount of outputAssetB
+     * @return isAsync Always false for this bridge
+     */
+    function convert(
+        AztecTypes.AztecAsset calldata inputAssetA,
+        AztecTypes.AztecAsset calldata inputAssetB,
+        AztecTypes.AztecAsset calldata outputAssetA,
+        AztecTypes.AztecAsset calldata outputAssetB,
+        uint256 totalInputValue,
+        uint256 interactionNonce,
+        uint64 auxData,
+        address rollupBeneficiary
+    )
+        external
+        payable
+        override(IDefiBridge)
+        returns (
+            uint256 outputValueA,
+            uint256 outputValueB,
+            bool isAsync
+        )
+    {
+        (bool enter, address underlyingAddress, address zkATokenAddress, bool isEth) = _sanityConvert(
+            inputAssetA,
+            inputAssetB,
+            outputAssetA,
+            outputAssetB
+        );
+
+        if (enter) {
+            outputValueA = _enter(underlyingAddress, zkATokenAddress, totalInputValue, isEth);
+        } else {
+            outputValueA = _exit(underlyingAddress, zkATokenAddress, totalInputValue, interactionNonce, isEth);
+        }
+
+        return (outputValueA, 0, false);
+    }
+
+    /**
+     * @notice Deposit into Aave with `amount` of `underlyingAsset` and return the corresponding amount of zkATokens
+     * @param underlyingAsset The address of the underlying asset
+     * @param amount The amount of underlying asset to deposit
+     * @return The amount of zkAToken that was minted by the deposit
```

Comment:
Description of zkATokenAddress and isEth is missing here.

Reply:
Should be added in https://github.com/AztecProtocol/aztec-connect-bridges/pull/21/commits/e017512e89b7177db72ba4e5e3b91e52e8c8f550